targets/targets.json

```diff
@@ -7727,7 +7730,8 @@
         "device_name": "nRF52840_xxAA",
         "macros_add": [
             "CONFIG_GPIO_AS_PINRESET",
-            "NRF52_ERRATA_20"
+            "NRF52_ERRATA_20",
+            "MBED_TICKLESS"
```

Comment:
This target inherits from MCU_NRF52840 which has MBED_TICKLESS

Reply:
Macro removed